### PR TITLE
Added Angular 2 tests and related updates

### PR DESCRIPTION
**Pull request by tomivirkki**
Connects to vaadin/components-team-tasks#126

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-combo-box/231)

### CODE REVIEW

File: directives/vaadin-combo-box.js [hunk 1]
```diff
@@ -63,6 +63,7 @@ System.register(['angular2/core', 'angular2/common', 'angular2/src/facade/lang']
                     }
                 };
                 VaadinComboBox.prototype.valuechanged = function (value) {
+                    var _this = this;
```
Comment (jouni) [inline directives/vaadin-combo-box.js]: Can’t we use ES5 `bind()` for these? Would look a bit cleaner IMO. But I don’t really care too much :)

Comment (tehapo) [inline directives/vaadin-combo-box.js]: This code is generated by the TypeScript compiler.

**Comment by platosha** [comment]
:lgtm:

---

Reviewed 10 of 10 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/231#-:-KGl5LJAR81paiRwLES7:1417088721)*

**Comment by manolo** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/231#-:-KGlRikDw80Hm0Gcoyqh:1417088721)*

**Comment by manolo** [comment]
Reviewed 10 of 10 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/231)*

**Pull request merged**